cloudsync/tasks.py

```diff
@@ -70,7 +70,7 @@ def stream_to_s3(self, video_id):
     try:
         video = Video.objects.get(id=video_id)
     except (Video.DoesNotExist, Video.MultipleObjectsReturned):
-        log.error("Exception retrieving video with id %d", video_id)
+        log.error("Exception retrieving video", video_id=video_id)
```

Comment:
One issue with the new approach is that a separate sentry error seems to be created for every distinct video_id.

For example, testing locally created these 2 separate errors in sentry:
https://sentry.io/organizations/mit-office-of-digital-learning/issues/1588586740/?project=194353
https://sentry.io/organizations/mit-office-of-digital-learning/issues/1588586642/?project=194353

in addition to creating this issue, with the 2 lumped together:
https://sentry.io/organizations/mit-office-of-digital-learning/issues/1588573250/events/?project=194353


whereas the current code just lumps them under the same error:
https://sentry.io/organizations/mit-office-of-digital-learning/issues/1588573222/events/?project=194353

Is there a way to configure this to avoid that?

Reply:
There should be an option in the Sentry configuration to tell it how to aggregate the errors. Might be worth looking at https://github.com/kiwicom/structlog-sentry

Reply:
I'll have a look at the Sentry settings.

The funny thing is that I thought this would deduplicate Sentry issues.

Reply:
I've just added the `structlog-sentry` module that @blarghmatey mentioned above. It's using the `SentryJsonProcessor`, which is supposed to prevent duplicate events, per https://github.com/kiwicom/structlog-sentry#logging-as-json ... @mbertrand, can you trigger another error so we can see what it looks like now in Sentry?